rgw/v2/tests/s3_swift/configs/test_bucket_policy_deny_actions.yaml

```diff
@@ -0,0 +1,40 @@
+# script: test_bucket_policy_ops.py
+# bucket policy deny actions
+# polarion id: CEPH-11216
```

Comment:
could you please update on the polarion id. As per the description expected CEPH-83593390

Reply:
@anrao19 
this is bucket policy testcase
not related to sts session policy polarion testcase
thats why I have added bucket policy polarion testcase only